fiblary3/common/event.py

```diff
@@ -38,7 +38,7 @@ class EventQueue(threading.Thread):
     def __init__(self, name=None):
         super(EventQueue, self).__init__(name=name or self.__class__.__name__)
         self.queue = queue.Queue()
-        self._stop = threading.Event()
+        self._stopper = threading.Event()
```

Comment:
`_stop` appears to be used internally by `threading.Thread` in recent versions of Python